code/Style.md

```diff
@@ -70,6 +72,28 @@ func Example() (err error) {
 }
 ```
 
+Sometimes, there are situations where a function, calls another function that only needs to return the value or the error returned by the call; when the called function return errors not created or wrapped by `errs` package the error must be  wrapped or annotated, however it isn't needed to have a conditional to achieve such purpose, because the functions that `errs` package have ignore the `nil` value when they are passed through; for example
+
+```go
+func OpenDataFile(filename string) (*os.File, error) {
+	file, err := os.Open(filepath.Join("data", filename))
+	if err != nil {
+		return nil, Error.Wrap(err)
+	}
+
+	return file, nil
+}
+```
+
+should be written like
+
+```go
+func OpenDataFile(filename string) (*os.File, error) {
+	file, err := os.Open(filepath.Join("data", filename))
+	return file, errs.Wrap(err)
```

Comment:
Shouldn't this be `Error.Wrap`?

Reply:
If there is one yes, but then in this snipped we have to add that `Error` definition otherwise it won't be clear.

Reply:
oh! I've just seen that I did that in the previous snipped.

Do you think that `Error` definition should be added to both?

Reply:
Yeah, I think so.